router.go

```diff
@@ -17,13 +17,29 @@
 package main
 
 import (
+	"encoding/json"
 	"net/http"
 
 	"github.com/gorilla/mux"
 )
 
+type HelloWorld struct {
+	Msg string `json:"msg"`
+}
+
 func setupRouter(router *mux.Router) {
 	// Setup your routes here.
 	router.HandleFunc("/", func(w http.ResponseWriter, req *http.Request) {
+		w.Header().Add("Content-Type", "application/json")
```

Comment:
This is a template, so we prefer do not insert code to remove after the service creation. But the idea it's great, could you create an example in another project with this hello world endpoint?

Reply:
Ofc, I removed it